sdk/js/src/svm.ts

```diff
@@ -82,6 +88,175 @@ export async function constructSubmitBidInstruction(
   return ixSubmitBid;
 }
 
+export function getAssociatedTokenAddress(
+  owner: PublicKey,
+  tokenMintAddress: PublicKey,
+  tokenProgram: PublicKey
+): PublicKey {
+  return getAssociatedTokenAddressSync(
+    tokenMintAddress,
+    owner,
+    true,
```

Comment:
makes sense why you defined this, but maybe a comment noting that this allows off-curve owners?